site/src/pages/WorkspacesPage/filter/filter.tsx

```diff
@@ -189,14 +189,19 @@ const StatusOptionItem = ({
 };
 
 const StatusIndicator: FC<{ option: StatusOption }> = ({ option }) => {
+  const color = option.color === "notice" ? "warning" : option.color;
```

Comment:
@aslilac I didn't figure out how to make it work with "notice" instead of "warning"